README.md

```diff
@@ -1,70 +1,150 @@
+```sh
+  git clone https://github.com/microhass/hello-react-front-end.git
+```
 
-### Advanced Configuration
+Navigate to the project directory:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+```sh
+ cd hello-react-front-end
+```
 
-### Deployment
+Install the necessary dependencies by running:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+```sh
+ npm install
+```
 
-### `npm run build` fails to minify
+Start the local dev server using the command:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+```sh
+  npm start
+```
+
+The website will be live at `localhost:3000/`. You can now make changes to the project as desired and push them back to the repository using Git.
+
+### Run tests
+
+To run tests, run the following command: -->
+
+
+
+```sh
+  npm test
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+<!-- > Mention all of the collaborators of this project. -->
+
+👤 **Hassan Shakur**
+
+- GitHub: [@hassanShakur](https://github.com/hassanShakur)
+- Twitter: [@hassShakur](https://twitter.com/hassShakur)
+- LinkedIn: [hassanShakur](https://www.linkedin.com/in/hassanShakur)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+If you'd like to contribute to this project, feel free to fork the repository and make changes as you see fit. Please submit a pull request with your changes and We'll review them as soon as possible.
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> We are excited to share with you my latest project on Github. As you know, Github is an essential tool for developers to showcase their skills, collaborate with others, and contribute to open-source projects. By supporting my project, you can help me improve us Github workflow skills and learn how to set up HTML & CSS linters.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> We would like to thank the Microverse community for their support and guidance throughout this project. Special thanks to our peers and mentors for their constructive feedback and advice.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- FAQ (optional) -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](MIT.md) licensed.
```

Comment:
- [ ] Great job working on this milestone and updating the ReadMe. Unfortunately, when I click on the MIT license link, it gives me an error 404 page. It is not professional to allow broken links in our codebase. Kindly fix this issue.

![Screen Shot 2023-10-26 at 8 45 19 PM](https://github.com/microhass/hello-react-front-end/assets/62849872/07812065-94b5-44b7-8506-41e408f42895)